x-pack/plugins/cases/server/services/configure/index.ts

```diff
@@ -85,7 +92,16 @@ export class CaseConfigureService {
         type: CASE_CONFIGURE_SAVED_OBJECT,
       });
 
-      return transformFindResponseToExternalModel(findResp);
+      const transformedConfigs = transformFindResponseToExternalModel(findResp);
+
+      const validatedConfigs: ConfigurationSavedObjectTransformed[] = [];
+      for (const config of transformedConfigs.saved_objects) {
+        const validatedAttributes = decodeOrThrow(ConfigurationAttributesRt)(config.attributes);
```

Comment:
nit: Maybe we can do `export type ConfigurationTransformedAttributesRt = ConfigurationAttributesRt;` in `x-pack/plugins/cases/server/common/types/configure.ts` and use it here instead of `ConfigurationAttributesRt`.